pages/[id].js

```diff
@@ -12,7 +12,9 @@ const StyledRecipeHeader = styled.div`
   justify-content: space-between;
 `;
 
-const BackLink = styled(Link)`
+const BackLink = styled(Link).attrs({
+  alt: "Back to recipes",
+})`
```

Comment:
a link does not receive an alt tag, but an aria-label. You can define it directly on the Link when you use it, this current method is quite hidden and hard to find if you want to change something in the future.